### DISCUSSION TITLE
Adonis equivalent of Laravel's Ziggy library

### DISCUSSION THREAD

**Discussion by eidellev** [adonisjs/core#3079] [Show and tell]
Hello Adonis community!
I just published [adonis-stardust](https://www.npmjs.com/package/@eidellev/adonis-stardust), which aims to emulate some of ziggy's functionality.

This is just v1 and hasn't been battle-tested yet, so I'd be happy to get some feedback as well as additional feature requests.

**Comment by thetutlage** [comment]
Great initiative. Can you please add it here as well? https://github.com/adonisjs-community/awesome-adonisjs

**Reply by eidellev** [reply]
Thank you! Will do :-)

**Comment by MrAhmedElkady** [comment]
Any updates for V6?

**Reply by RomainLanz** [reply]
There is Tuyau by @Julien-R44. You can sponsor him to have access to the documentation: https://julr.dev/posts/introducing-my-sponsorship-program
